src/go/rpk/pkg/cli/shadow/config.go

```diff
@@ -120,7 +120,7 @@ func generateSampleConfig() *ShadowLinkConfig {
 		},
 		ConsumerOffsetSyncOptions: &ConsumerOffsetSyncOptions{
 			Interval: 30 * time.Second,
-			Enabled:  true,
+			Paused:   true,
```

Comment:
nit: I think people will expect to have this enabled by default.
i.e. most people will generate the file and adjust the client options and ignore the rest, so better to give them something that works by default, even if it's an example.